packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -45,18 +45,19 @@ export const zoomMeetingsSchema = z.object({
 });
 
 const zoomTokenSchema = z.object({
-  scope: z.string().regex(new RegExp("meeting:write")),
-  expiry_date: z.number(),
+  scope: z.string().regex(new RegExp("meeting:write")).optional(),
+  expiry_date: z.number().or(z.null()),
   expires_in: z.number().optional(), // deprecated, purely for backwards compatibility; superseeded by expiry_date.
-  token_type: z.literal("bearer"),
-  access_token: z.string(),
-  refresh_token: z.string(),
+  token_type: z.literal("bearer").optional(),
+  access_token: z.string().optional(),
+  refresh_token: z.string().optional(),
```

Comment:
Schema changes are due to the error response from Zoom not containing these fields

Reply:
Let's make a `z.union` so we can receive two schemas. The one where the zoom response is successful and the schema where we don't have `scope` and other fields instead of optional everything